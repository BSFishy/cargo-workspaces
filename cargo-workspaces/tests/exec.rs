mod utils;
#[cfg(unix)]
use insta::assert_snapshot;

#[cfg(unix)]
#[test]
fn test_normal() {
    let (out, err) = utils::run("../fixtures/normal", &["ws", "exec", "cat", "Cargo.toml"]);
    assert_snapshot!(out);
    assert_snapshot!(err);
}
