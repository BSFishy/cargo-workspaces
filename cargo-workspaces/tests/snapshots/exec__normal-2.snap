---
source: tests/exec.rs
expression: err
---
info success ok

